docs/DEVELOPMENT.md

```diff
@@ -83,7 +83,7 @@ grpc.reflection.v1alpha.ServerReflection
 tekton.results.v1alpha2.Results
 
 # Makes a request to the Results service.
-$ grpc_cli call --channel_creds_type=ssl --ssl_target=tekton-results-api-service.tekton-pipelines.svc.cluster.local --call_creds=access_token=$(kubectl get secrets -n tekton-pipelines -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='tekton-results-debug')].data.token}"|base64 --decode) localhost:50051 tekton.results.v1alpha2.Results.ListResults 'parent: "default"'
+$ grpc_cli call --channel_creds_type=ssl --ssl_target=tekton-results-api-service.tekton-pipelines.svc.cluster.local --call_creds=access_token=$(kubectl create token tekton-results-debug -n tekton-pipelines) localhost:50051 tekton.results.v1alpha2.Results.ListResults 'parent: "default"'
```

Comment:
I would suggest breaking this command into two, since the token is gonna be used multiple times, it would be nicer to save it in a variable and then reference it. WDYT?

`export SOME_TOKEN_NAME=$(kubectl create token tekton-results-debug -n tekton-pipelines)`

Reply:
I see this more as a demo code, but if people find it useful then why not :-). Updated.